### PR TITLE
feat: show reset step button contexually

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This pr shows the reset button along with the ask for help button at the bottom of the lower jaw. 
Ref #46178

Screen Shot 2022-06-01 at 5 23 20 PM

### CODE REVIEW

File: client/src/templates/Challenges/classic/lower-jaw.tsx [hunk 1]
```diff
@@ -159,19 +161,25 @@ const LowerJaw = ({
       : sentenceArray[0];
   };
 
-  const renderHelpButton = () => {
+  const renderLowerLowerJaw = () => {
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/lower-jaw.tsx:164]:
```suggestion
  const renderContexualActionRow = () => {
```

File: client/src/templates/Challenges/classic/lower-jaw.tsx [hunk 2]
```diff
@@ -214,6 +221,7 @@ const LowerJaw = ({
       >
         {renderTestFeedbackContainer()}
       </div>
+      {renderLowerLowerJaw()}
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/lower-jaw.tsx:224]:
```suggestion
      {renderContexualActionRow()}
```

**Review by ShaunSHamilton — DISMISSED** [review]
I am happy with this, but are we sure:
1) The option only exists after failing to pass the tests a few times
2) To have the reset button so close to the _Ask for Help_ button

I might suggest adding back the "Are you sure..." modal.

**Comment by Sembauke** [comment]
Hey @ahmadabdolsaheb.

I think @ShaunSHamilton has a valid point here, another solution would be adding a divider between the two buttons as they are not correlated at all. 

Another option would just be adding more space between the buttons.

**Comment by ahmadabdolsaheb** [comment]
I agree that spacing would be helpful here. Since the button now pops a modal and the button appears after a set of attempts, the likelihood of a using making a mistake is highly reduced.

**Review by ShaunSHamilton — APPROVED** [review]
Brilliant, Ahmad! 👍

**Review by Sembauke — APPROVED** [review]

**Pull request merged**